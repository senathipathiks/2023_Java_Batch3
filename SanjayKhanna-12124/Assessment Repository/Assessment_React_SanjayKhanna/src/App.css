.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #1d1f23;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: rgb(255, 253, 253);
  
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}

.page-container {
  display: flex;
  flex-direction: column;
  height: 100vh;
}

.content-container {
  flex: 1;
  padding: 1rem;
}

