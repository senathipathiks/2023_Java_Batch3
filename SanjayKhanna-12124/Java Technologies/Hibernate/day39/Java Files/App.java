package com.sms.mavenday1.Student;

import org.hibernate.Session;
import org.hibernate.SessionFactory;
import org.hibernate.Transaction;
import org.hibernate.cfg.Configuration;

import com.sms.maven.excerise.Employee;





/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
//        System.out.println( "Hello World!" );
    	
//    	Student s1 = new Student(1, "Sanjaykhanna", "Chennai");
    	Employee e1 = new Employee(12124, "Sanjay Khanna", 23);
    	Configuration cfg = new Configuration().configure("hibernate.cfg.xml");
    	SessionFactory factory = cfg.buildSessionFactory();
    	Session session = factory.openSession();
    	Transaction tans = session.getTransaction();
    	tans.begin();
    	session.save(e1);
    	tans.commit();
    	
    	
    	
    }
}
