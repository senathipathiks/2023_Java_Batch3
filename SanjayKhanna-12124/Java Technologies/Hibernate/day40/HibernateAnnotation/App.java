package com.sms.mavan.HibernateAnnotation;

import org.hibernate.Session;
import org.hibernate.SessionFactory;
import org.hibernate.Transaction;
import org.hibernate.cfg.Configuration;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
    	Configuration cfg = new Configuration().configure("hibernate.cfg.xml");
    	SessionFactory factory = cfg.buildSessionFactory();
    	Session session = factory.openSession();
    	Transaction tans = session.getTransaction();
    	
    	Student stu1 = new Student();
    	stu1.setStuName("prasath");
    	stu1.setCity("Chennai");
    	
    	tans.begin();
    	session.save(stu1);
    	tans.commit();
    }
}
