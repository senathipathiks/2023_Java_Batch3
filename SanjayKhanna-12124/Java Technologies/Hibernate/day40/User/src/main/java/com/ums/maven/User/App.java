package com.ums.maven.User;

import java.util.ArrayList;
import java.util.Scanner;

import com.ums.bean.User;
import com.ums.dao.UserDAO;

public class App 
{
	private static Scanner sc = new Scanner(System.in);
	private static UserDAO dao = new UserDAO();
	
	public static void main( String[] args )
    {
    	boolean decision = true;
    	System.out.println("Welcome to User Management System\n-----------------------------------------------------");
        do {
        	System.out.println("1-Add Record\n2-Delete Record\n3-Update Record\n4-Search Record\n5-View All Records\n6-Quit\nEnter your option : ");
        	int op = sc.nextInt();
        	switch(op) {
        	case 1:{
        		User u = getUser();
        		boolean n= dao.insertUser(u);
        		if(n)
        			System.out.println("Record Inserted Successfully");
        		else
        			System.out.println("Insertion Failure");
        		break;
        	}
        	case 2:{
        		int id = getId();
        		boolean n = dao.deleteUser(id);
        		if(n)
        			System.out.println("Record Deleted Successfully");
        		else
        			System.out.println("Deletion Failure");
        		break;
        	}
        	case 3:{
        		User u = getUser();
        		boolean n = dao.updateUser(u);
        		if(n)
        			System.out.println("Record Update Successfully");
        		else
        			System.out.println("Updation Failure");
        		break;
        	}
        	case 4:{
        		int id = getId();
        		User u = dao.searchUser(id);
        		if(u != null) 
        			System.out.println("Record Found !\n"+u.toString());
        		else
        			System.out.println("Record not found");
        		break;
        	}
        	case 5:{
        		ArrayList<User> u = dao.viewall();
        		if(u==null) {
        			System.out.println("No record found");
        		}
        		else {
        			System.out.println("Total no of Record : "+u.size());
        			for(User us : u)
        				System.out.println(us.toString());
        		}
        		break;
        	}
        	case 6:{
        		System.out.println("Are you Confirm?[y/n]");
        		String ch = sc.next();
        		if(ch.charAt(0)=='y')
        			decision = false;
        		else
        			System.out.println("Quitting process failure");
        		break;
        	}
        	default: System.out.println("Invalid Option \nPlease enter only 1-6");
        	}
        	
        	System.out.println("-------------------------------------------------\n");
        	
        }while(decision);
        System.out.println("Thank you for using");
    }
	
	private static User getUser() {
		System.out.println("Enter User Id, Username, and Password");
		return new User(sc.nextInt(), sc.next(), sc.next());
	}
	
	private static int getId() {
		System.out.println("Enter User Id:");
		return sc.nextInt();
	}
}
