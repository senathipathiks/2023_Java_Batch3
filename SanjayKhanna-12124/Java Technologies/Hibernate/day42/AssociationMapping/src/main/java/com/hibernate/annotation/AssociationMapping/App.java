package com.hibernate.annotation.AssociationMapping;

import java.util.ArrayList;

import java.util.List;

import org.hibernate.Session;
import org.hibernate.SessionFactory;
import org.hibernate.Transaction;
import org.hibernate.cfg.Configuration;

//import com.sample.bean.Author;
//import com.sample.bean.Book;
import com.sample.bean.Customer;
import com.sample.bean.Product;


//import com.sample.bean.Department;
//import com.sample.bean.Student;


public class App 
{
    public static void main( String[] args )
    {
        Configuration cfg = new Configuration().configure("hibernate.cfg.xml");
        SessionFactory factory = cfg.buildSessionFactory();
        Session session = factory.openSession();
        Transaction trans = session.getTransaction();
        
//        trans.begin();
//        Laptop l = new Laptop(1, "Dell");
//        session.persist(l);
//        Employee emp = new Employee(1, "Sanjay",l);   
//        session.persist(emp);
//        trans.commit();
        
//        Department d1 = new Department(1, "CSE");
//        Department d2 = new Department(2, "IT");
//        Student s1 = new Student(1, "Sanjay", d1);
//        Student s2 = new Student(2, "Sam", d1);
//        Student s3 = new Student(3, "Praveen", d2);
//        
//        trans.begin();
//        session.persist(s1);
//        session.persist(s2);
//        session.persist(s3);
//        trans.commit();
        
//        Book b1 = new Book(1, "Harry Potter part 1");
//        Book b2 = new Book(2,"Harray Potter part 2");
//        Book b3 = new Book(3,"Ponniyan Selvan");
//        List<Book> list = new ArrayList<Book>();
//        list.add(b1);
//        list.add(b2);
//        List<Book> l2 = new ArrayList<Book>();
//        l2.add(b3);
//        Author a1 = new Author(1, "Kalki", l2);
//        Author a2 = new Author(2, "J.K Rowling", list);
//        trans.begin();
//        session.persist(a1);
//        session.persist(a2);
//        trans.commit();
//        session.close();
        
        Product p1 = new Product(1, "Smart Phone");
        Product p2 = new Product(2, "Laptop");
        Product p3 = new Product(3, "Television");
        Product p4 = new Product(4, "Air Conditioner");
        
        List<Product> l1 = new ArrayList<Product>();
        l1.add(p1);
        l1.add(p2);
        l1.add(p3);
        
        List<Product> l2 = new ArrayList<Product>();
        l2.addAll(l2);
        l2.add(p4);
        
        Customer c1 = new Customer(1, "SanjayKhanna", l2);
        Customer c2 = new Customer(3, "Dharshsun", l1);
        Customer c3 = new Customer(2, "Vignesh", l1);
        
        trans.begin();
        session.persist(c1);
        session.persist(c2);
        session.persist(c3);
        trans.commit();
        session.close();
    }
}
