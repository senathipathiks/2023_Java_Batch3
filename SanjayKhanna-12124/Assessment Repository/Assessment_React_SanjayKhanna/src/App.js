import { useState } from 'react'
import { BrowserRouter, Link, Route, Routes } from 'react-router-dom'
import './App.css'
import Home from './Home'
import LaptopCreate from './LaptopCreate'
import LaptopDetail from './LaptopDetail'
import LaptopEdit from './LaptopEdit'
import LaptopListing from './LaptopListing'
import LaptopLogin from './LaptopLogin'
import LaptopRegister from './LaptopRegister'


function App()
{
  const [navbarOpen, setNavbarOpen] = useState(false)

  return (
    <BrowserRouter>
      <div className="page-container">
        <nav className="navbar navbar-expand-lg bg-body-tertiary ">
          <div className="container"
          >
            <button
              className="navbar-toggler ms-auto"
              type="button"
              data-bs-toggle="collapse"
              data-bs-target="#navbarNav"
              aria-controls="navbarNav"
              aria-expanded="false"
              aria-label="Toggle navigation"
              onClick={() => setNavbarOpen(!navbarOpen)}
            >
              <span className="navbar-toggler-icon"></span>
            </button>
            <div
              className={`collapse navbar-collapse ${navbarOpen ? 'show' : ''}`}
              id="navbarNav"
            >
              <ul className="navbar-nav ms-auto mb-2 mb-lg-0"
              >
                <li className="nav-item"
                >
                  <Link className="nav-link" to="/">Home</Link>
                </li>
                <li className="nav-item"
                >
                  <Link className="nav-link" to="/laptop/login">Login</Link>
                </li>
                <li className="nav-item"
                >
                  <Link className="nav-link" to="/laptop/Register">Register</Link> 
                </li>
              </ul>
            </div>
          </div>
        </nav>
        <div className="container-fluid"
        >
          <Routes>
            <Route path="/" element={<Home />}></Route>
            <Route path="/laptop/login" element={<LaptopLogin />}></Route>
            <Route path="/laptop/listing" element={<LaptopListing/>}></Route>

            <Route path="/laptop/create" element={<LaptopCreate />}></Route>
            <Route path="/laptop/detail/:lapId" element={<LaptopDetail />}></Route>
            <Route path="/laptop/edit/:lapId" element={<LaptopEdit />}></Route>
            <Route path="/laptop/Register" element={<LaptopRegister />}></Route> 
          </Routes>
        </div>
      </div>
    </BrowserRouter>
  )
}

export default App