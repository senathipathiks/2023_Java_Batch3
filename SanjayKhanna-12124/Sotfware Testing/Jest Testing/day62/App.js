import './App.css';

function App() {
  return (
    <div className="App">
      <h1>Welcome to my page</h1>
      <br/>
      <h1>This is Sanjay khanna S</h1>
      <h2>Software Trainee Engineer</h2>
    </div>
  );
}

export default App;
