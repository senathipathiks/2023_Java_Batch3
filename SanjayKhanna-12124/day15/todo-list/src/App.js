import { Provider } from 'react-redux';
import './App.css';
import TodoApp from './Component/TodoApp';
import store from './configure-store';

function App() {
  return (
    <div> 
    <Provider store={store}>
    <TodoApp />
  </Provider>
  </div>
   
  );
}

export default App;
