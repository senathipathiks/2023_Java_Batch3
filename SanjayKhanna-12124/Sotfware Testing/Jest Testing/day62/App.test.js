import renderer from 'react-test-renderer';
import App from './App';

// describe('Test Suite for App',()=>{
//   test('renders learn react line 1', () => {
//     render(<App />);
//     const linkElement = screen.getByText("Welcome to my page");
//     expect(linkElement).toBeInTheDocument();
//   });
//   test('renders learn react line 2', () => {
//     render(<App />);
//     const linkElement = screen.getByText("This is Sanjay khanna");
//   expect(linkElement).toBeInTheDocument();
//   });
//   test('renders learn react line 3', () => {
//     render(<App />);
//     const linkElement = screen.getByText("Software Trainee Engineer");
//     expect(linkElement).toBeInTheDocument();
//   });
// })

test('Snapshot Testing', () => {
  const comp = renderer.create(<App />).toJSON();
  expect(comp).toMatchSnapshot();
});

