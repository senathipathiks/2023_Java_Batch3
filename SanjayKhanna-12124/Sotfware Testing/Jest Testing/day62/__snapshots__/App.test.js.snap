// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing 1`] = `
<div
  className="App"
>
  <h1>
    Welcome to my page
  </h1>
  <br />
  <h1>
    This is Sanjay khanna S
  </h1>
  <h2>
    Software Trainee Engineer
  </h2>
</div>
`;
